search/search_with_filtering.py

```diff
@@ -30,14 +30,14 @@ def get_search_service_client():
 
 
 # get search service request:
-def get_search_request(query: str, _filter: str, page_size=10):
+def get_search_request(query: str, _filter: str):
     default_search_placement = "projects/" + project_number + "/locations/global/catalogs/default_catalog/placements/default_search"
 
     search_request = SearchRequest()
     search_request.placement = default_search_placement  # Placement is used to identify the Serving Config name.
     search_request.query = query
     search_request.filter = _filter
-    search_request.page_size = page_size
+    search_request.page_size = 10
```

Comment:
are you hard coding values to match tests?

Reply:
no, just to limit the products in the output, 10 products are fully displayed in the Terminal